phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -7,13 +7,6 @@ abstract class Package_PHP_XHTML extends Package_Generic_XHTML {
         'appendix'              => 'format_container_chunk',
         'article'               => 'format_container_chunk',
         'book'                  => 'format_root_chunk',
-        'classname'             => array(
-            /* DEFAULT */          'span',
-            'ooclass'           => array(
-                /* DEFAULT */      'format_suppressed_tags',
-                'classsynopsisinfo' => 'format_classsynopsisinfo_ooclass_classname',
-            ),
-        ),
```

Comment:
For reference, this is useless, as it duplicated the same information from the generic XHTML format.